recipe/meta.yaml

```diff
@@ -30,22 +30,19 @@ requirements:
     #- datasets !=2.5.0
```

Comment:
`transformers-cli` is an entry_point above. What was the intention of commenting `datasets` out? It seems that without `datasets` the entry_point is broken, right?
If we do not provide datasets then it will be pip installed and not conda installed, see https://github.com/huggingface/transformers/blob/2782aadae2b0b0c313eac3ee70f84f0335577635/src/transformers/utils/import_utils.py#L109

Reply:
I'm not certain what the intention was, as the change was made for a previous version, but I've added it back in.